kiwi/builder/live.py

```diff
@@ -302,20 +302,26 @@ def create(self) -> Result:
         else:
             # Put the root filesystem into SquashFS directly
             with FileSystem.new(
-                name='squashfs',
+                name=root_filesystem,
                 device_provider=DeviceProvider(),
                 root_dir=self.root_dir + os.sep,
                 custom_args={
                     'compression':
                         self.xml_state.build_type.get_squashfscompression()
-                }
+                } if root_filesystem == 'squashfs' else {'compression': ''}
```

Comment:
Can we not set erofs compression?

Reply:
We have a `squashfscompression` attribute and so far none for erofs. I go with the default setting in the initial support. In a followup PR we can add custom erofs settings as we see fit

Reply:
If you look at `man mkfs.erofs` there are many options to set for the filesystem. I'd like to have a conversation first which of those are a priority for our use cases

Reply:
Okay. I'd definitely like to see us expose the ability to configure erofs compression, because otherwise livecds with erofs will be larger than with squashfs...

Reply:
Here's what I see for erofs compression options:

```
 -zX[,level=Y]         X=compressor (Y=compression level, Z=dictionary size, optional)
    [,dictsize=Z]      alternative compressors can be separated by colons(:)
    [:...]             supported compressors and their option ranges are:
                         lz4
                         lz4hc
                           [,level=<0-12>]		(default=9)
                         lzma
                           [,level=<0-9,100-109>]	0-9=normal, 100-109=extreme (default=6)
                           [,dictsize=<dictsize>]	(default=<auto>, max=8388608)
                         deflate
                           [,level=<0-9>]		(default=1)
                           [,dictsize=<dictsize>]	(default=32768, max=32768)
                         libdeflate
                           [,level=<0-12>]		(default=1)
                         zstd
                           [,level=<0-22>]		(default=3)
                           [,dictsize=<dictsize>]	(default=<auto>, max=1048576)
```

The ones I expect to use are zstd, lz4/lz4hc, or lzma. I don't really expect to use deflate any. I also expect exposing the compression level setting too.

Reply:
ok I added the `erofscompression` attribute